java/opendcs/src/test/java/decodes/tsdb/alarm/mail/AlarmMailerTest.java

```diff
@@ -0,0 +1,84 @@
+package decodes.tsdb.alarm.mail;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Properties;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+import com.icegreen.greenmail.util.GreenMail;
+import com.icegreen.greenmail.util.ServerSetup;
+import decodes.tsdb.alarm.AlarmGroup;
+import decodes.tsdb.alarm.EmailAddr;
+import decodes.sql.DbKey;
+
+
+public class AlarmMailerTest {
+
+	private static GreenMail smtpServer;
+	private AlarmMailer alarmMailer;
+	private Properties props;
+    private final String fromAddr = "karl@opendcs.org";
```

Comment:
I would use the standard example.org, or whatever the "valid" non-existent domain is. If we ever did setup email for the domain, we're going to get a lot of junk at some point.